controller/getTopic.controller.js

```diff
@@ -1,7 +1,6 @@
 const { fetchTopic } = require("../model/fetchTopic.model");
 
 exports.getTopic = (req, res) => {
-  console.log("in get toppic controller");
   fetchTopic().then((data) => {
```

Comment:
try to get into the habit of naming your parameters clear names

`data` isn't very clear - `topics` would be better here